server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/Serializer.scala

```diff
@@ -209,39 +210,44 @@ class Serializer @Inject() (mailboxIdFactory: MailboxId.Factory) {
   private implicit val notFoundWrites: Writes[NotFound] = Json.valueWrites[NotFound]
   private implicit val mailboxGetResponseWrites: Writes[MailboxGetResponse] = Json.writes[MailboxGetResponse]
 
-  def serialize(session: Session): JsValue = {
-    Json.toJson(session)
-  }
+  private implicit val jsonValidationErrorWrites: Writes[JsonValidationError] = error => JsString(error.message)
 
-  def serialize(requestObject: RequestObject): JsValue = {
-    Json.toJson(requestObject)
-  }
+  private implicit def jsonValidationErrorsWrites(implicit jsonValidationErrorWrites: Writes[JsonValidationError]): Writes[JsonSeq[JsonValidationError]] =
+    (errors: JsonSeq[JsonValidationError]) => {
```

Comment:
did you try to use `Iterable` to avoid this painful types?

Reply:
Nope, but that's worth a shot ! Let me check

Reply:
Compiler seems to complain in this case, pity